src/components/RestaurantList/RestaurantList.jsx

```diff
@@ -0,0 +1,31 @@
+import "./RestaurantList.css";
+
+function RestaurantList({ restaurants }) {
+  return (
+    <section className="restaurant-list-container">
+      <ul className="restaurant-list">
+        {restaurants.map((restaurant) => (
```

Comment:
```suggestion
        {restaurants.map(({ id, image, category, name, description }) => (
```
로건이 목에서 피날 정도로 강의해줬던 [구조 분해 할당](https://ko.javascript.info/destructuring-assignment) 문법을 사용해보는건 어떨까요??🙃

Reply:
아유 너무 좋네요..... 로건의 목소리가 귀에 맴도네요 적용해보도록 하겠슴둥 ‼